CHANGELOG.rst

```diff
@@ -1,5 +1,60 @@
 .. _changelog:
 
+0.46.0
+------
+
+This release introduces universal cloud storage, allowing users to mount storage from various providers such as S3 or WebDAV in sessions, based on RClone.
+It also adds the ability to modify the resource class for a session that is either hibernated or 
+failing.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- 🔌 **UI**: add support for more storage services
+  (`#2908 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2908>`_,
+  `#2915 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2915>`_).
+
+**✨ Improvements**
+
+- 🖌️ **UI**: Improve the look and feel of the home page
+  (`#2968 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2968>`_,
+  `#2937 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2937>`_,
+  `#2927 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2927>`_).
+- 🔐 **UI**: Use password fields for credentials
+  (`#2920 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2920>`_).
+- **CLI**: Allow mounting cloud storage in sessions launched from the CLI 
+  (`#3629 <https://github.com/SwissDataScienceCenter/renku-python/issues/3629>`__).
+- **Core Service**: return date_published in datasets.list response
+  (`#3648 <https://github.com/SwissDataScienceCenter/renku-python/issues/3648>`__). 
+
+**🐞 Bug Fixes**
+
+- **UI**: Resize the feedback badge on the session settings page
+  (`#2953 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2953>`_).
+- **UI**: Fix the environment dropdown on the Start session page
+  (`#2949 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2949>`_).
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**Improvements**
+
+- **UI**: Add more features to the admin page to manage resource classes
+  (`#2916 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2916>`_,
+  `#2910 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2910>`_).
```

Comment:
These are helpful features! I think it's worth naming what they are.

```suggestion
- **UI**: RenkuLab admins can now add tolerations and node affinities to resource classes
  (`#2916 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2916>`_).
- **UI**: RenkuLab admins can add multiple users to a resource pool at once via a list of emails.
  `#2910 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2910>`_).
```

Also, could you please move this to the User-Facing Changes section? Admins are still users.